tests/robottelo/test_settings.py

```diff
@@ -0,0 +1,81 @@
+"""Unit tests for sample json settings"""
+import json
+import os
+
+from unittest2 import TestCase
+
+JSON_INVALID_ITEMS = ['hostname ', ' scheme', 'None', '']
```

Comment:
This is just an initial set of invalid test data that I could think of.  We can add more whenever necessary :)